Promote Fether tutorial

Tbaut opened the pull request:
Mention and link to the wiki Fether tutorial

yjkimjunior commented on README.md:
```diff
@@ -1,17 +1,12 @@
 # Parity Signer - Turn your smartphone into a hardware wallet
 
-Parity Signer is a mobile application that turns a smartphone into a secure air-gapped wallet also called cold storage. It means that, once the app has been installed, users can create Ethereum accounts, sign transactions and transfer funds from and to these accounts without any sort of connectivity enabled on the device. Wifi, Mobile Network, Bluetooth can be shut down and even removed physicall ...
+Parity Signer is a mobile application that turns a smartphone into a secure air-gapped wallet also called cold storage. It means that, once the app has been installed, users can create Ethereum accounts, sign transactions and transfer funds from and to these accounts without any sort of connectivity enabled on the device. Wifi, Mobile Network, Bluetooth can be shut down and even removed physicall ...
```
I know this wasn't your wording to begin with but I've made some grammar edits, for what it's worth:
### -
Parity Signer is a mobile application that allows any smartphone to act as a secure, air-gapped crypto wallet. This is also known as "cold storage." 
 
Once the app has been installed, users can create Ethereum accounts, sign transactions and transfer funds to and from these accounts without any sort of connectivity enabled on the device. 

Users can opt to shut off and even physically remove the smartphone's Wifi, Mobile Network, and Bluetooth to ensure that the mobile phone containing these accounts will not be exposed to any online threat. 

Have a look at the tutorial on our wiki to learn how to use [Parity Signer together with Fether](https://wiki.parity.io/Parity-Signer-Mobile-App-Fether-tutorial) or [MyCrypto app](https://wiki.parity.io/Parity-Signer-Mobile-App-MyCrypto-tutorial).

maciejhirsz replied:
To be fair, we should stress that while disabling networking is technically an option, it is pretty much a requirement for the app to be used as intended.

Tbaut commented:
Thanks, I've replaced the text by the one you suggested @yjkimjunior and added a bold statement to stress out the need to disable any networking ability.

yjkimjunior commented on README.md:
```diff
@@ -6,10 +6,18 @@
 
 Any data transfer from or to the app will happen using QR code scanning. By doing so, the most sensitive piece of information, the private keys, will never leave the phone. The Parity Signer mobile app can be used to store any Ethereum account, this includes ETH, ETC as well as Ether from various testnets (Kovan, Ropsten...).
 
+**Disabling the mobile phone's networking abilities is a requirement for the app to be used as intended.**
```
suggestion: could we move this up two paragraphs so it comes right after the sentence `Users can opt to shut off and even physically ...`

yjkimjunior commented on README.md:
```diff
@@ -6,10 +6,18 @@
 # Parity Signer - Turn your smartphone into a hardware wallet
 
-Parity Signer is a mobile application that turns a smartphone into a secure air-gapped wallet also called cold storage. It means that, once the app has been installed, users can create Ethereum accounts, sign transactions and transfer funds from and to these accounts without any sort of connectivity enabled on the device. Wifi, Mobile Network, Bluetooth can be shut down and even removed physicall ...
+Parity Signer is a mobile application that allows any smartphone to act as an air-gapped crypto wallet also known as "cold storage."
```
suggestion: `...air-gapped crypto wallet. This is also known as...`

yjkimjunior commented on README.md:
```diff
@@ -6,10 +6,18 @@
-Parity Signer is a mobile application that turns a smartphone into a secure air-gapped wallet also called cold storage. It means that, once the app has been installed, users can create Ethereum accounts, sign transactions and transfer funds from and to these accounts without any sort of connectivity enabled on the device. Wifi, Mobile Network, Bluetooth can be shut down and even removed physicall ...
+Parity Signer is a mobile application that allows any smartphone to act as an air-gapped crypto wallet also known as "cold storage."
+
+Once the app has been installed, users can create Ethereum accounts, sign messages, transactions and transfer funds to and from these accounts without any sort of connectivity enabled on the device.
```
suggestion: `...create Ethereum accounts, sign messages/transactions, and transfer funds to and from these...``

yjkimjunior commented:
Looks good to me :)
